docs/source/overview/features.rst

```diff
@@ -8,20 +8,32 @@ Filter out your target from the background with HSV (Hue, Saturation, Value) and
 
 Offset Calibration
 ^^^^^^^^^^^^^^^^^^^^
-More often than not, it is not possible and/or inconvenient to place the camera in the center of the robot. Using special Chameleon Vision tools, it is easy to make a software fix for the physical offset from the true center.
 
+Oftentimes, it is difficult to place the camera in the direct center of the robot.
+Chameleon Vision contains special tools to fix the incorrect positioning.
 
-Widespread Camera Support
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Chameleon Vision supports a large family of :ref:`USB cameras<supported-cameras>`. This allows you to get the most flexibility compared to other solutions. The Chameleon Vision Team recommends the PS3ye Camera (found on Amazon and other vendors) for maximum performance. 
 
-.. note:: The Pi Camera and all network cameras are both NOT currently supported.
+Wide Range of Supported Hardware
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Chameleon Vision supports a wide range of both :ref:`coprocessors<supported-coprocessors>` and :ref:`cameras<supported-cameras>`.
+This allows for the maximum amount of flexibility in both performance and cost.
```

Comment:
Awkward wording, maybe `best/utmost flexibility`

Reply:
Neither of these sound the best to me.

Reply:
a greater degree of flexibility?